index.md

```diff
@@ -66,9 +66,8 @@ if it wasn't for the OSG. Sharing allows individual researchers to access larger
   </div>
   <div class="col-lg-4">
     <h3>Find Us!</h3>
-    <p>Are you a user wanting more computing resources?</p>
-    <p>Are you a resource provider wanting to join our collaboration?</p>
-    <p>If so, find us at the <a href="https://support.opensciencegrid.org">support desk</a>.</p>
+    <p>Are you a resource provider wanting to join our collaboration? Contact us: <a href="mailto:support@opensciencegrid.org">support@opensciencegrid.org</a>.</p>
+    <p>Are you a user wanting more computing resources? Check with your 'local' computing providers, or consider using <a href="https://www.osgconnect.net/">OSG Connect</a> (available to US-based academic/govt/non-profit research projects).</p>
```

Comment:
Should we steal from #76 and add a line here that says something like:

```suggestion
    <p>Are you a user wanting more computing resources? Check with your 'local' computing providers, or consider using <a href="https://www.osgconnect.net/">OSG Connect</a> (available to US-based academic/govt/non-profit research projects).</p>
    <p>For any other inquiries, reach us at: <a href="mailto:help@opensciencegrid.org">help@opensciencegrid.org</a>.</p>
```

Reply:
yes, but it needs to be support@opensciencegrid.org !

Reply:
Oops, fixed